pkg/text/text.go

```diff
@@ -76,5 +76,5 @@ func ContinueTask(s string, cont, noConfirm bool) bool {
 
 	response = strings.ToLower(response)
```

Comment:
In this case you should not need the ToLower if you're using `strings.EqualFold`, at least from checking the documentation